accounts.txt

```diff
@@ -0,0 +1,152 @@
+one1y0xcf40fg65n2ehm8fx5vda4thrkymhpg45ecj
```

Comment:
really need to have another file with account list?  can you just grep the account list from internal/genesis/harmony.go directly?